spec/requests/api/v1/collections_spec.rb

```diff
@@ -46,124 +55,167 @@
     context 'a required parameter is empty/blank' do
       before do
         invalid_params = {}
-        post "/api/v1/collections", params: invalid_params, headers: valid_session
+        post '/api/v1/collections', params: invalid_params, headers: valid_session
       end
 
       it 'returns failure message as JSON' do
         expect(response.status).to eq(400)
-        expect(JSON.parse(response.body)).to match(hash_including('status' => 400, "developer_message" => "handle is missing, handle is empty, token is missing, token is empty"))
+        expect(JSON.parse(response.body)).to match(
+          hash_including('status' => 400,
+                         'developer_message' => 'handle is missing, handle is empty, token is missing, token is empty')
+        )
       end
     end
 
     context 'handle uses illegal characters' do
       before do
-        invalid_params = { "handle" => "agency-blogs", "token" => "secret" }
-        post "/api/v1/collections", params: invalid_params, headers: valid_session
+        invalid_params = { 'handle' => 'agency-blogs', 'token' => 'secret' }
+        post '/api/v1/collections', params: invalid_params, headers: valid_session
       end
```

Comment:
While we are cleaning up can we avoid using hash rockets on areas that are getting cleaned up.

Reply:
👍